ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/IntegrationTestUtil.java

```diff
@@ -0,0 +1,205 @@
+package io.confluent.ksql.rest.integration;
+
+import com.google.common.base.Preconditions;
+import io.confluent.ksql.KsqlExecutionContext;
+import io.confluent.ksql.test.util.EmbeddedSingleNodeKafkaCluster;
+import io.confluent.ksql.util.PersistentQueryMetadata;
+import io.confluent.ksql.util.QueryMetadata;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.apache.kafka.common.IsolationLevel;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.streams.KafkaStreams.State;
+import org.apache.kafka.streams.TopologyDescription;
+import org.apache.kafka.streams.TopologyDescription.Source;
+import org.apache.kafka.streams.TopologyDescription.Subtopology;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class IntegrationTestUtil {
```

Comment:
you just moved this logic right, i couldnt find any differences in the logic itself from a quick look

Reply:
Yeah, just cut and pasted it.  No intended changes.